setup.py

```diff
@@ -64,8 +64,8 @@ def run_tests(self):
     raise Exception('this library does not support python >2 and <3.5')
```

Comment:
Actually, the error message was correct.  The suggested `this library does not support python >3 and <3.5` mean it doesn't support version 4 or less than 3.5, which isn't right.

I'm happy to have the error string that uses real English too :)